client/src/Components/App.js

```diff
@@ -1,13 +1,25 @@
 import React, { Component } from 'react';
+import { BrowserRouter,Route} from 'react-router-dom';
+import Dashboard from './Pages/Dashboard';
+import Hero from './Pages/LandingPage/HeroSection';
+import Header from './CommonComponents/Header';
+import Footer from './CommonComponents/Footer'
 
 import './App.css';
 
 class App extends Component {
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-        </header>
+       <Header />
+      <BrowserRouter>
+       <div>
+       <Route exact path="/" component={Hero} />
+       <Route path="/dash" component={Dashboard}/>
+       </div>
+      </BrowserRouter>
+      <Footer />
```

Comment:
This looks good.  I have a few small suggestions (which aren't required, but might help in the future) -

1) Move the `<BrowserRouter>` tag up one level, so that it wraps `<div className="App">`.  Then the BrowserRouter wraps the Header and Footer too, which means Header and Footer has router information available to it (so for example the header could have a NavLink which properly gets "active" styles when you're on a certain route).

2) Wrap your list of routes in a `<Switch>` (see [here](https://reacttraining.com/react-router/core/api/Switch)).  When `<Switch>` is around a bunch of routes, it will only show one of those routes - the first route that matches the current url - and there's no danger of creating two routes that both match the current url and accidentally showing both of those components.  This also means, for example, that you can add a route below all of your other routes to "catch" any url that doesn't match, and show a 404 page.  For example:
```
<Switch>
  <Route exact path="/' component={Home} />
  <Route path="/jobs/:job" component={JobDetails} />
  <Route path="/jobs" component={JobList} /> {/* Without the switch, this would also show up when the user visited /jobs/123, which would be bad */}
  <Route component={my404Page} /> {/* Since we have a switch, this only matches if none of the routes above it match */}
</Switch>
```
In general, all of our App.js files have their list of routes wrapped in a switch statement for these reasons.